lib/datum/version.rb

```diff
@@ -1,9 +1,10 @@
+# frozen_string_literal: true
+
 module Datum
   # @!visibility private
-  VERSION = "4.3.1"
+  VERSION = '4.3.3'
```

Comment:
I see, you had to change this each time to trigger the package build workflow

Reply:
yeah since it's basing on these lines here:
https://github.com/syntax-engineering/datum/blob/df048a5ae35158412eeb7236317e79a7c7d507dd/.github/workflows/release_gem_package.yml#L2-L8

every time time we push a commit to the `master` branch, that contains changes to the `version.rb` file, we should run this workflow

(it you're testing on a PR, it's fine to comment out lines 5-6, so it only looks at when the file changes)